src/meta/app/src/schema/table.rs

```diff
@@ -616,3 +616,47 @@ pub struct CountTablesReply {
 pub struct TableIdToName {
     pub table_id: u64,
 }
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Eq, PartialEq, Default)]
+pub struct TableCopiedFileNameIdent {
+    pub tenant: String,
+    pub db_id: u64,
+    pub table_id: u64,
+    pub file: String,
+}
```

Comment:
Does a file name identity need `tenant` or `db_id`?

In my opinion, it is a table-specific concept.